packages/ui-react-utils/src/passthroughProps.ts

```diff
@@ -48,13 +50,16 @@ const omit = (originalObject: Record<string, unknown>, keys: string[]) => {
   return newObject
 }
 
-function omitProps(
-  props: Record<string, any>,
-  propTypes: Record<string, any>,
+function omitProps<T extends Record<string, any>>(
```

Comment:
Now that you know how these utils work exactly, could you please add a few lines of description (js annotations) above them? :) Just so the IDE can help if we hover over the names.

Reply:
Actually, there are separate files (pickProps.ts, omitProps.ts...) next to this... Probably for the doc page to generate pages for them, but then the code should be separated too, what do you think?

Reply:
Done